recipes/aiida-fleur/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "aiida-fleur" %}
+{% set version = "1.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiida-fleur-{{ version }}.tar.gz
+  sha256: 7d15f99ca1c340110655826ad0bbc36f0c48743377fa5522f8448dc6787d9f05
+
+build:
+  number: 0
+  skip: True  # [win or py < 3.6]
```

```suggestion
  noarch: python
```

nice pkg name; this is a good candidate for noarch btw, let's try that?